src/custom-blocks/accordion/index.php

```diff
@@ -88,10 +88,13 @@ function render_callback_accordion_block_section($attributes, $content)
             </h3>
         </div>
         <div id="accordion-default-content-1" class="govuk-accordion__section-content" aria-labelledby="accordion-default-heading-1">
-            <p class="govuk-body">
-                <?php _e(esc_html($attribute_accordion_section_TextArea)) ; ?>
-                <?php _e(esc_html($content)); ?>
-            </p>
+            <?php
+            $firstParagraphContent = _(trim(esc_html($attribute_accordion_section_TextArea)));
+            if (!empty($firstParagraphContent)) {
+                echo "<p class='govuk-body'>$firstParagraphContent</p>";
+            }
```

Comment:
Should  _e(esc_html($content));  be in a p tag?

Reply:
No, as it could be a list or other block (I think just a list actually, but it might not be a p)